hipcheck/src/engine.rs

```diff
@@ -68,8 +64,12 @@ fn default_query_explanation(
 ) -> Result<Option<String>> {
 	let core = db.core();
 	let key = get_plugin_key(publisher.as_str(), plugin.as_str());
-	let Some(p_handle) = core.plugins.get(&plugin) else {
-		return Err(hc_error!("No such plugin {}", key));
+	let Some(p_handle) = core.plugins.get(&key) else {
+		return Err(hc_error!(
+			"No such plugin {}, {:?}",
+			key,
+			core.plugins.keys().collect::<Vec<&String>>()
```

Comment:
Do we really want to be listing all of the running plugins in the error message? This seems more like something for `log::debug()`

Reply:
That is left over debugging code. I will update that tomorrow!

Reply:
I'd like to get this merged in today, could you just remove it quickly? Otherwise I'll merge the branch as-is and you can submit a fix as a separate PR.

Reply:
I can push something in a couple of minutes!